### PR TITLE
chore: update skyux-dev-infra and nx

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1977/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/a11y-storybook/)
- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/colorpicker-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/core-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/modals-storybook/)
- [navbar-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/navbar-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/phone-field-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/popovers-storybook/)
- [progress-indicator-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/progress-indicator-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/tiles-storybook/)
- [toast-storybook](https://blackbaud.github.io/skyux-pr-preview/1977/storybooks/toast-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1977/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1977/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1977/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1977/playground/)

File: libs/components/i18n/schematics/ng-generate/lib-resources-module/files/__name@dasherize__-resources.module.ts.template [hunk 1]
```diff
@@ -23,7 +22,10 @@ import {
 SkyLibResourcesService.addResources(RESOURCES);
 
 export class <%= classify(name) %>ResourcesProvider implements SkyLibResourcesProvider {
-  public getString(localeInfo: SkyAppLocaleInfo, name: string): string | undefined {
+  public getString(
+    localeInfo: SkyAppLocaleInfo,
+    name: string,
+  ): string | undefined {
```
Comment (johnhwhite) [inline libs/components/i18n/schematics/ng-generate/lib-resources-module/files/__name@dasherize__-resources.module.ts.template:28]: These changes are necessary to satisfy the "check resources" job after code formatting runs.

File: libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @use 'libs/components/theme/src/lib/styles/mixins' as mixins;
 @use 'libs/components/theme/src/lib/styles/variables' as *;
-@use 'node_modules/intl-tel-input/build/css/intlTelInput';
+@import 'node_modules/intl-tel-input/build/css/intlTelInput.css';
```
Comment (johnhwhite) [inline libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.scss:3]: Switching to `@import` to [load this file as CSS (not SASS)](https://sass-lang.com/documentation/at-rules/import/), which fixes an issue with `esbuild` and resolving the background image `url()`.

File: libs/components/lookup/src/lib/modules/country-field/country-field.component.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 @use 'libs/components/theme/src/lib/styles/mixins' as mixins;
 @use 'libs/components/theme/src/lib/styles/variables' as *;
-@use 'node_modules/intl-tel-input/build/css/intlTelInput';
+@import 'node_modules/intl-tel-input/build/css/intlTelInput.css';
```
Comment (johnhwhite) [inline libs/components/lookup/src/lib/modules/country-field/country-field.component.scss:3]: Switching to `@import` to [load this file as CSS (not SASS)](https://sass-lang.com/documentation/at-rules/import/), which fixes an issue with `esbuild` and resolving the background image `url()`.

File: apps/code-examples/project.json [hunk 4]
```diff
@@ -82,8 +82,8 @@
       "executor": "@nx/eslint:lint",
       "options": {
         "lintFilePatterns": [
-          "apps/code-examples/src/**/*.ts",
-          "apps/code-examples/src/**/*.html"
+          "{projectRoot}/src/**/*.ts",
+          "{projectRoot}/src/**/*.html"
```
Comment (johnhwhite) [inline apps/code-examples/project.json:86]: These changes were an NX schematic. Every `project.json` file with a `lint` target has this small change.

File: apps/code-examples/src/app/code-examples/ag-grid/data-entry-grid/basic/data.ts [hunk 5]
```diff
@@ -22,7 +22,7 @@ export const DEPARTMENTS = [
   },
 ];
 
-export const JOB_TITLES: { [name: string]: AutocompleteOption[] } = {
+export const JOB_TITLES: Record<string, AutocompleteOption[]> = {
```
Comment (johnhwhite) [inline apps/code-examples/src/app/code-examples/ag-grid/data-entry-grid/basic/data.ts:25]: These changes are a new eslint rule `@typescript-eslint/consistent-indexed-object-style`.

File: apps/code-examples/src/app/code-examples/forms/radio/icon/demo.component.ts [hunk 6]
```diff
@@ -8,11 +8,11 @@ import {
 } from '@angular/forms';
 import { SkyRadioModule } from '@skyux/forms';
 
-type Item = {
+interface Item {
```
Comment (johnhwhite) [inline apps/code-examples/src/app/code-examples/forms/radio/icon/demo.component.ts:11]: Types were changed to interfaces by a new eslint rule `@typescript-eslint/consistent-type-definitions`.

File: apps/code-examples/src/app/code-examples/lists/filter/modal/filter-modal.component.ts [hunk 7]
```diff
@@ -64,13 +64,13 @@ export class FilterModalComponent {
   }
 
   #setFormFilters(appliedFilters: Filter[]): void {
-    for (let i = 0; i < appliedFilters.length; i++) {
-      if (appliedFilters[i].name === 'fruitType') {
-        this.fruitType = `${appliedFilters[i].value}`;
+    for (const appliedFilter of appliedFilters) {
```
Comment (johnhwhite) [inline apps/code-examples/src/app/code-examples/lists/filter/modal/filter-modal.component.ts:67]: Iteration loops were updated due to a new eslint rule `@typescript-eslint/prefer-for-of`.

File: apps/e2e/a11y-storybook-e2e/src/e2e/skip-link.component.cy.ts [hunk 8]
```diff
@@ -11,13 +11,9 @@ describe('a11y-storybook', () => {
...
-        cy.get('#ready')
-          .should('exist')
-          .end()
-          .get('.sky-skip-link')
-          .should('exist')
-          .focus()
-          .should('be.visible');
+        cy.get('#ready').should('exist');
+        cy.get('.sky-skip-link').should('exist').focus();
+        cy.get('.sky-skip-link').should('be.visible');
```
Comment (johnhwhite) [inline apps/e2e/a11y-storybook-e2e/src/e2e/skip-link.component.cy.ts:16]: A new Cypress eslint rule recommends against chaining after certain interactions such as `click()`, `focus()`, and `screenshot()`.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**